### PR TITLE
13861 - Update SBC-Auth to SBC-common-components 3.0.3 to support BC Reg Staff.

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:*
https://github.com/bcgov/entity/issues/13861

*Description of changes:*
Update sbc-common-components

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by seeker25** [comment]
image

File: auth-web/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "moment": "^2.29.4",
     "regenerator-runtime": "^0.13.3",
     "register-service-worker": "^1.6.2",
-    "sbc-common-components": "3.0.1",
+    "sbc-common-components": "3.0.3",
```
Comment (severinbeauvais) [inline auth-web/package.json:33]: Maybe bump up line 3 as well (even though that version is not exposed yet).

File: auth-web/src/App.vue [hunk 2]
```diff
@@ -3,6 +3,7 @@
     <div class="header-group" ref="headerGroup">
       <sbc-loader :show="showLoading" />
       <sbc-header
+        class="flex-direction-column"
```
Comment (severinbeauvais) [inline auth-web/src/App.vue:6]: Would one of these work instead?

https://vuetifyjs.com/en/styles/flex/#flex-direction

**Review by severinbeauvais — APPROVED** [review]
Whatever you decide in OK by me.

File: auth-web/src/App.vue (hunk 2 above)
Comment (seeker25) [reply auth-web/src/App.vue:6]: .flex-column

Comment (seeker25) [reply auth-web/src/App.vue:6]: let me bookmark this page -^

File: auth-web/package.json (hunk 1 above)
Comment (seeker25) [reply auth-web/package.json:33]: should be good

**Pull request merged**